app/templates/emails/g-cloud_application_started.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+   <meta charset="UTF-8">
+</head>
+<body>
+    Dear supplier,
+    <br/><br/>
+    You, or one of your team, started an application to become a {{ framework.name }} supplier.
+    All contributors on your Digital Marketplace account will now receive email updates.
+    The deadline for completing your application is {{ framework_dates.framework_close_date }}.
+    <br/><br/>
+    You must ask any clarification questions you have through your Digital Marketplace account before
+    {{ framework_dates.clarifications_close_date }}.
+    <br/><br/>
+    To complete your application, you must:
+    <ul>
+      <li>complete your company details</li>
+      <li>make the supplier declaration</li>
+      <li>provide information about the services you offer, and mark each service as complete</li>
+    </ul>
+    <br/><br/>
+    To understand if your services are suitable for {{ framework.name }}, read this guidance:
+    <br/>
+    <a href="https://www.gov.uk/guidance/g-cloud-suppliers-guide">https://www.gov.uk/guidance/g-cloud-suppliers-guide</a>
+    <br/><br/>
+    The {{ framework.name }} application section of your Digital Marketplace account helps you:
```

Comment:
The content linked in the Trello ticket shows that this should have a non breaking space. I don't think the name from the framework does. Could use the nbsp filter. 

Having said that, it's not likely that it's ever going to be an issue here so you may not need to bother.

Reply:
Good spot. Added.